orc8r/tools/packer/magma-focal-virtualbox-patch-upgrade-ovs-1.1.20210618.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Is it possible to add a license header to this file?

Reply:
no, json does not allow comments